src/content/docs/alerts-applied-intelligence/notifications/destinations.mdx

```diff
@@ -0,0 +1,64 @@
+  - Destinations
+metaDescription: "Read about how to configure and set up Incident Intelligence destinations."
+---
+
+<Callout title="Early access">
+The features described here are early access.
+
+For more information on related features, see our docs on [Alerts notification channels](/docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/notification-channels-control-where-send-alerts/), [Incident Intelligence destinations](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/incident-intelligence-destination-examples/), and [Proactive Detection notifications](https://docs.newrelic.com/docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/proactive-detection-applied-intelligence/#set-up).
+</Callout>
+
+Destinations are unique identifiers for third-party systems. The destination you choose is the location where we send your notifications. Destination settings contain connection details to integrate with third-party systems and can be used across a variety of tools in New Relic One.
+
+The supported destination platforms include:
+
+* Email
+* Jira
+* ServiceNow
+* Slack
+* Webhook
+
+For more on these and other destinations, see [notification integrations](/docs/alerts-applied-intelligence/notifications/notification-integrations).
+
+<Callout variant="tip">
+  It's also possible to configure destinations using the `aiNotifications` [NerdGraph  API](https://developer.newrelic.com/collect-data/get-started-nerdgraph-api-explorer/).
+</Callout>
+
+## Required roles in New Relic One [#required-roles]
+
+Destinations settings require specific user capabilities.
+
+* To access your settings: you need `View` capabilities for `Applied Intelligence:Destinations` or `Alerts`.
+* To modify or delete your settings: you need `Modify` capabilities for `Applied Intelligence:Destinations` or `Alerts`.
+
+## Add a destination [#add-destination]
+
+Follow these steps to add a new destination for your notifications.
+
+1. Go to [one.newrelic.com](https://docs.newrelic.com/one.newrelic.com), click **Alerts & AI**, and, in the left nav under **Enrich and Respond**, click **Destinations**.
+
+2. Select one of the destination types at the top of the destination management screen.
+
+3. The **Destinations** table shows information about the existing destinations and allows users to enable, disable, modify, or create new destinations.
+
+![A screenshot of the destinations overview page.](./images/destinations-overview.png "The destinations overview page")
+
+<figcaption>If you have access to it, find the destinations overview page at **one.newrelic.com > Alerts & AI > Destinations**.</figcaption>
+
+To add a destination, click the appropriate platform tile. To modify destination settings, click the destination row in the destinations table
+
+## Notifications log [#notifications-log]
+
+To view past notification events details, go to the **Destination** menu, and click the **Notifications Log** tab.
```

Comment:
Can we encourage the team to downcase the L in logs? Unless a UI writer has written it that way. But it's my understanding that we use sentence case in the UI unless it's a proper noun, and this shouldn't be a proper noun.

Reply:
An oversight on my part. I'll continue to encourage sentence casing there :)